ideascube/configuration/registry.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+
+
+# Please try and keep this stored in alphabetical order
+REGISTRY = {
+    # 'namespace': {
+    #     'key': {
+    #         'python_type': str,
+    #         'default': 'Default value',
+    #     },
+    # },
+    'server': {
+        'site-name': {
+            'python_type': str,
```

Comment:
I renamed it `type` in the latest push to this PR.

I'll call the future one `pretty_type` instead.